packages/widgets-manager/src/reducers/__tests__/mounted_apps.spec.ts

```diff
@@ -0,0 +1,104 @@
+import mountedAppReducer, { initialState } from '../mounted_apps';
+import {
+  syncData,
+  mountAppSuccess,
+  unMountAppSuccess,
+} from '../../store/actions';
+
+describe('When it recieves syncData action', () => {
+  it('Should set the received mounted_apps', () => {
+    const mockMountedApps = {
+      1: true,
```

Comment:
Do we always identify apps by an integer? Do we have a mapping to the app-name somewhere?